php/thankYou.php

```diff
@@ -122,45 +305,61 @@
                 echo "<p>T-shirt Size: $tShirt</p>";
 
                 //address
-                echo "<p>Address: ".$street." ".$city.", ".$state." ".$zip."</p>";
+                echo "<h5>Address</h5>";
+                echo "<p>Street: $street</p>";
+                echo "<p>City: $city</p>";
+                echo "<p>State: $state</p>";
+                echo "<p>Zip Code: $zip</p>";
 
                 //interests
                 echo "<h5>Interests</h5>";
-                if(!isset($_POST["interests"])){
-                    echo "<p>None selected</p>";
-                }
-                else {
-                    echo "<ul>";
-                    foreach($_POST["interests"] as $interests) {
-                        if($interests === "other"){
-                            echo "<li>Other Skills: ".$_POST["interestsText"]."</li>";
-                            $mailInterests .= ", {$_POST["interestsText"]}";
-                        }
-                        else {
-                            echo "<li>" . $interests . "</li>";
-                            $mailInterests .=  ", $interests";
+                    if (!isset($_POST["interests"])) {
+                        echo "<p>None selected</p>";
+                    }
+                    else if(!validSelectWText($interstArray,$_POST["interests"])){
+                        echo "<p>Illegal Selection</p>";
+                        $isValid = false;
+                    }
+                    else {
+                        echo "<ul>";
+                        foreach ($_POST["interests"] as $interests) {
+                            if ($interests === "other" && validText($_POST["interestsText"])) {
+                                echo "<li>Other Interest: " . $_POST["interestsText"] . "</li>";
+                                $mailInterests .= ", {$_POST["interestsText"]}";
+                            }
+                            else if ($interests === "other" && validText($_POST["interestsText"]) && trim($_POST["interestsText"]) === "" ) {
+                                echo "<li>Other</li>";
+                                $mailInterests .= ", {$_POST["interestsText"]}";
+                            }
```

Comment:
if  the condition ' trim($_POST["interestsText"]) === "" ' is true then ' $mailInterests .= ", {$_POST["interestsText"]}"; ' will put nothing into $mailInterests except for ', ' everytime. We could since nothing is typed in we could just use $interests since we know it says 'other'.